pkg/db/validator.go

```diff
@@ -19,19 +19,19 @@ func NewValidator(blockQueries bool, logger *zap.SugaredLogger) *Validator {
 }
 
 func (v *Validator) Validate(query string) error {
-	matchSelect, err := regexp.MatchString("SELECT.*(FROM\\s*\\w+\\s*)+(WHERE (\\w*\\s*=\\s*\\$\\d+(\\s*,\\s*)?(\\s+AND\\s+)?(\\s+OR\\s+)?)*)?(\\w*\\s+IN\\s+[^;]+)?(\\w*\\s+ORDER BY\\s+[^;]+)?(\\w*\\s+GROUP BY\\s+[^;]+)?$", query)
+	matchSelect, err := regexp.MatchString("SELECT.*(FROM\\s*\\w+\\s*)+(WHERE (\\(?\\w*\\s*[=<>]\\s*\\$\\d+\\)?(\\s*,\\s*)?(\\s+AND\\s+)?(\\s+OR\\s+)?)*)?(\\w*\\s+IN\\s+[^;]+)?(\\w*\\s+ORDER BY\\s+[^;]+)?(\\w*\\s+GROUP BY\\s+[^;]+)?$", query)
```

Comment:
Without this change, a few requests would not work. For example, queries with `<` or `>` instead of `=`. Additionally, I found a bug that causes that built queries like [here](https://github.com/kyma-incubator/reconciler/pull/487/files#diff-b70a62f2599a781583b295541d8528116e619114a361834082e2651a18234581R16) would not pass validation because of brackets